### PR TITLE
Additions to EngiVend and Vendomat inventory

### PR DESCRIPTION
**Pull request by crystalHex**
## About the PR

This PR adds the holofan projector and geiger counter to the EngiVend and glass beakers to the Vendomat.

## Why / Balance

The holofan and geiger counter should really be accessible alongside the other tools.
The glass beakers are required for some of the machines in the CircuitVend, and setting up an autolathe just to print 2 beakers seems a bit inconvenient.

## Technical details

YML

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: The EngiVend now sells geiger counters and holofan projectors.
- tweak: The Vendomat now sells glass beakers.

### CODE REVIEW

**Comment by dvir001** [comment]
Beaker is a bit of a wild card one but not a big issue.
GeigerCounter alright, remove the need to buy a full crate

HolofanProjector not sure about, but not a real issue also

Can you take a picture with the in game prices, or list them?

**Comment by crystalHex** [comment]
image
I have not modified these prices from their base values yet.
It seems like the vending machines take the StaticPrice of the item and multiply it by 10 to get the sell price.

**Review by dvir001 — APPROVED** [review]

**Review by FoxxoTrystan — APPROVED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/engivend.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@
     Multitool: 10
     ClothingEyesGlassesMeson: 10
     ClothingHeadHatWelding: 6
+    HolofanProjector: 10    # Frontier
```
Comment (Cheackraze) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/engivend.yml:8]: Holofan is a bit much, I think we should still restrict them.

Comment (dvir001) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/engivend.yml:8]: They exists on ships plenty as of now, also not much abused as they main use is redoing ship walls for building.

**Comment by dvir001** [comment]
@crystalHex To get this added for now please remove the HolofanProjector and ill approve this PR, after that you can make a new PR just for the holo so we can reconsider also adding it.

**Comment by dvir001** [comment]
Remved the ``HolofanProjector`` so we can add this change for now.

**Pull request merged**